config/scripts/validate-tokens.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+yarn build-tokens > /dev/null
+
+if (git diff --quiet src/generated > /dev/null); then
+  echo "Design-tokens are up to date."
```

Comment:
isn't this then/else inverted? If there's no diff then the exit code of `git diff` is `0` - shouldn't that trigger the `else` rather than the `then`?

Reply:
`git diff --quiet` exit code is 0 when there is no diff with means it enters the first if close (no error)
And the exit code is 1, which means error/failure, and that's assimilated to false to it enters the else.

Reply:
it's not the same as returning a boolean true or false or a number, it's an exit code, the exit code value doesn't really matter in this case. You could read it as `if no error in git diff`.

Reply:
OK, that's confusing :/ - thanks for the explanation!